lib/components/src/syntaxhighlighter/syntaxhighlighter.tsx

```diff
@@ -155,15 +155,7 @@ export const SyntaxHighlighter: FunctionComponent<Props> = ({
       </Scroller>
 
       {copyable ? (
-        <ActionBar
-          actionItems={[
-            {
-              title: copied ? 'Copied' : 'Copy',
-              className: `copy-button ${copied ? 'copy-button--copied' : ''}`,
-              onClick,
-            },
-          ]}
-        />
```

Comment:
You decided these classnames were not really needed?

Reply:
Yes because it was more about the code part itself which is not directly related to an SB block! It will not solve #14673